Update `TabView` to latest MUX sources

MartinZikmund opened the pull request:
GitHub Issue (If applicable): related to  #9730, closes https://github.com/unoplatform/uno/issues/21062, closes #4255, closes https://github.com/unoplatform/uno/issues/4829, closes https://github.com/unoplatform/uno/issues/21232

<!-- Link to relevant GitHub issue if applicable. All PRs should be associated with an issue (GitHub issue or internal), unless the change is documentation related. -->

## PR Type

What kind of change does this PR introduce?
<!-- Please uncomment one or more that apply to this PR

- Bugfix
- Feature
- Code style update (formatting)
- Refactoring (no functional changes, no api changes)
- Build or CI related changes
- Documentation content changes
- Project automation
- Other... Please describe:

-->

## What is the current behavior?

<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->


## What is the new behavior?

<!-- Please describe the new behavior after your modifications. -->


## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below.
     Please note that breaking changes are likely to be rejected -->

## Other information

<!-- Please provide any additional information if necessary -->

Internal Issue (If applicable):
<!-- Link to relevant internal issue if applicable. All PRs should be associated with an issue (GitHub issue or internal) -->

MartinZikmund committed:
chore: Start updating TabView to latest MUX sources

MartinZikmund committed:
chore: File-scoped namespaces

MartinZikmund committed:
chore: Update TabViewItem to latest MUX source

MartinZikmund committed:
feat: TabView upgrade to latest MUX sources

MartinZikmund committed:
chore: Update TabView styles

MartinZikmund committed:
feat: Add support for ProviderFromPeer

MartinZikmund committed:
test: Update TabView runtime tests

MartinZikmund committed:
chore: Adjust for CI

ajpinedam reviewed (approved):
Nice!

jeromelaban commented:
@MartinZikmund is this PR still valid?

MartinZikmund commented:
> @MartinZikmund is this PR still valid?

Finalizing it now

MartinZikmund committed:
chore: Bump TabView to MUX 5f52761

MartinZikmund committed:
chore: Bump TabView to MUX c8d3b4a

MartinZikmund committed:
chore: Adjust for CI

MartinZikmund committed:
chore: Adjust test page

MartinZikmund committed:
chore: Add back Uno-specific adjustments

MartinZikmund committed:
chore: Adjust for invalid XAML

jeromelaban commented:
@MartinZikmund There are two tests not passing on android.

Youssef1313 commented on src/Uno.UI/UI/Xaml/Automation/Peers/AutomationPeer.cs:
```diff
@@ -106,6 +106,8 @@ public partial class AutomationPeer : DependencyObject
 
 		public AutomationPeer GetLabeledBy() => GetLabeledByCore();
 
+		protected internal IRawElementProviderSimple ProviderFromPeer(AutomationPeer peer) => new IRawElementProviderSimple(peer);
```
@MartinZikmund The method is redefined in the same file. Could you fix that and see if this will get green?

MartinZikmund committed:
chore: Adjut workarounds

jeromelaban commented:
/azp run

MartinZikmund committed:
chore: Update TabView.h to winiu3/release/1.6.1

MartinZikmund committed:
chore: Update TabView.properties to winui3/release/1.6.1

MartinZikmund committed:
chore: Update TabViewListView to winui3/release/1.6.1

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/index.html

morning4coffe-dev committed:
chore: Update `TabView` to winui 1.7

morning4coffe-dev committed:
chore: Adjust `TabView` tests

morning4coffe-dev committed:
chore: Update `TabViewItem` to winui 1.7

morning4coffe-dev committed:
chore: Misc `TabView` adjustments

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/index.html

MartinZikmund committed:
chore: Update

MartinZikmund committed:
chore: Adjust usings

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-10576/index.html

morning4coffe-dev committed:
chore: Add missing using for UWP build

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-10576/index.html

morning4coffe-dev committed:
chore: More UWP namespace fixes

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-10576/index.html

morning4coffe-dev committed:
chore: More minor adjustments

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-10576/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/index.html

morning4coffe-dev committed:
chore: Multiple CI fixes

morning4coffe-dev committed:
chore: Adjust unused events

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/index.html

morning4coffe-dev committed:
feat: Implement missing args

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/index.html

morning4coffe-dev committed:
feat: Implement `MoveSizeOperation`

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/index.html

morning4coffe-dev commented on src/Uno.UI/Microsoft/UI/Xaml/Controls/TabView/TabView.cs:
```diff
@@ -29,1351 +28,2462 @@
+	private const double c_scrollAmount = 50.0;
 
-		public TabView()
+	public partial class TabViewTabTearOutWindowRequestedEventArgs : EventArgs
```
@MartinZikmund In WinUI source those are part of TabView.cpp as well. I think it isn't supposed to be use anywhere else even in the future?

MartinZikmund replied:
They are public members which live in the namespace directly, not inside the type - otherwise people would have to refer to them as `TabView.TabViewTabTearOutWindowRequestedEventArgs` in the code, and it would not be in line with WinUI. The types themselves are in `TabView.cpp`/`.h`, but not inside the type itself - they just decided to declare them in the same file:

You can see here - both event args and `TabView` are classes on the same level (the event args are not inside the `TabView` class):

![image](https://github.com/user-attachments/assets/2bce71e7-7b33-46b5-a0c5-ec0454dfcee3)
![image](https://github.com/user-attachments/assets/ffc8f76e-6cd9-4b9f-b5f5-4591d7871452)

morning4coffe-dev committed:
chore: Add semi-implement more windowing APIs

morning4coffe-dev committed:
chore: TabView adjustments

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-10576/index.html

morning4coffe-dev committed:
chore: More adjustments

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-10576/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/index.html

morning4coffe-dev committed:
chore: Adjust `TabViewTests` namespaces

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-10576/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/docs/index.html

MartinZikmund committed:
chore: Duplicate member

MartinZikmund committed:
chore: Adjust focus manager usages

MartinZikmund committed:
chore: Adjust syntax

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-10576/wasm-skia-net9/index.html

MartinZikmund committed:
chore: Separate out event args

MartinZikmund committed:
chore: Resources merge

MartinZikmund committed:
chore: Re-add missing constructors

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/docs/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-10576/wasm-skia-net9/index.html

MartinZikmund committed:
test: Behavior correction for existing tests

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/docs/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-10576/wasm-skia-net9/index.html

MartinZikmund committed:
chore: Undo missing API changes for now

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/docs/index.html

MartinZikmund committed:
chore: Resolve build issues

MartinZikmund committed:
chore: Adjust property

MartinZikmund committed:
chore: Adjust

MartinZikmund committed:
fix: Correct `CornerRadius` theme resources

The `CornerRadius_themeresources` should be included in the `XamlControlsResources`, but not in "system" resources, otherwise the `HasKey` is incorrectly reporting them as being present in any control's resource dictionary.

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-10576/wasm-skia-net9/index.html

MartinZikmund committed:
fix: Call PrepareContainerForIndex for all items on Android

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-10576/docs/index.html

unodevops commented:
⚠️⚠️ The build [173793](https://dev.azure.com/uno-platform//Uno%20Platform/_build/results?buildId=173793) has failed on Uno.UI - CI.

MartinZikmund committed:
chore: Final cleanup

MartinZikmund committed:
chore: Disable test on native iOS

nventive-devops commented:
The [build 173919](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=173919) found UI Test snapshots differences: `android-28-net8`: **26**, `android-28-net8-Snap`: **55**, `ios`: **6**, `ios-Snap`: **53**, `skia-linux-screenshots`: **81**, `skia-windows-screenshots`: **290**, `wasm`: **75**, `wasm-automated-net9.0-WinUI-Benchmarks-automated`: **0**, `wasm-automated-net9.0-WinUI-Default-automated`: **20**, `wasm-automated-net9.0-WinUI-RuntimeTests-0`: **0**, `wasm-automated-net9.0-WinUI-RuntimeTests-1`: **0**, `wasm-automated-net9.0-WinUI-RuntimeTests-2`: **0**

<details>
<summary>Details</summary>


* `android-28-net8`: **26** changed over 825

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Button_NestedButtons_Validation_nVentive_Umbrella_Views_UI_Samples_Content_UITests_ButtonTestsControl_Nested_Buttons`
  - `DecimalFormatterTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `Detereminate_ProgressRing_Validation25_[#FF0000_#008000_#FF0000_#FF0000]_Progress-Ring-Value-25`
  - `Detereminate_ProgressRing_Validation75_[#FF0000_#008000_#008000_#008000]_Progress-Ring-Value-75`
  - `ListView_SelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `LoadEmptyContentControl_ContentControlNoTemplateNoContent_-_bntContentClear`
  - `NativeCommandBar_Size_Uno_UI_Samples_Content_UITests_CommandBar_CommandBar_Dynamic`
  - `LoadEmptyContentControl_SamplesApp_Windows_UI_Xaml_Controls_ContentControlNoTemplateNoContent`
  - `SequentialAnimations_SamplesApp_Windows_UI_Xaml_Media_Animation_SequentialAnimationsPage`
  - `TextBox_TextChanging_Capitalize_UITests_Shared_Windows_UI_Xaml_Controls_TextBoxTests_TextBox_TextChanging`
  - `TextBox_TextChanging_Limit_UITests_Shared_Windows_UI_Xaml_Controls_TextBoxTests_TextBox_TextChanging`
  - `When_Bound_To_GlobalThemedResources_UITests_Shared_Windows_UI_Xaml_ThemeResources_AppXamlDefinedResources`
  - `When_Click_Outside_Dialog_Expect_No_Dismiss_UITests_Shared_MessageDialogTests_MessageDialogTest`
  - `When_ExtendedSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_SingleSelectionWithItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_SingleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `UpDownEnabledTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `LoadEmptyContentControl_CContentControl`
  - `SelectionContentTest_UITests_Microsoft_UI_Xaml_Controls_TabViewTests_TabViewItemsSourceTests`
  - `When_MultipleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  </details>

* `android-28-net8-Snap`: **55** changed over 1075

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Gesture_Recognizer_Pointer_Events_test_bench_UITests_Shared_Windows_UI_Input_GestureRecognizer_PointersEvents`
  - `Brushes_GenericApp_Views_Content_UITests_ImageBrushTestControl_ImageBrush_UniformToFill_GenericApp_Views_Content_UITests_ImageBrushTestControl_ImageBrush_UniformToFill`
  - `ComboBox_ComboBox_NoDataContext_UITests_Shared_Windows_UI_Xaml_Controls_ComboBox_ComboBox_NoDataContext`
  - `ListView_ListViewSelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `NavigationView_MUXControlsTestApp_NavigationViewRS4Page_MUXControlsTestApp_NavigationViewRS4Page`
  - `Animations_DoubleAnimation_RenderTransformOrigin_GenericApp_Views_Content_UITests_Animations_DoubleAnimation_RenderTransformOrigin`
  - `ComboBox_ComboBox_ToggleDisabled_UITests_Windows_UI_Xaml_Controls_ComboBox_ComboBox_ToggleDisabled`
  - `MediaPlayerElement_Mini_player_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Minimal`
  - `Image_UITests_Windows_UI_Xaml_Controls_ImageTests_SvgImageSource_Basic_UITests_Windows_UI_Xaml_Controls_ImageTests_SvgImageSource_Basic`
  - `Image_UITests_Windows_UI_Xaml_Controls_ImageTests_SvgImageSource_FromMsAppData_UITests_Windows_UI_Xaml_Controls_ImageTests_SvgImageSource_FromMsAppData`
  - `Image_UITests_Windows_UI_Xaml_Controls_ImageTests_SvgImageSource_FromStream_UITests_Windows_UI_Xaml_Controls_ImageTests_SvgImageSource_FromStream`
  - `SwipeControl_MUXControlsTestApp_SwipeControlPage2_MUXControlsTestApp_SwipeControlPage2`
  - `Animations_DoubleAnimation_Opacity_TextBlock_GenericApp_Views_Content_UITests_Animations_DoubleAnimation_Opacity_TextBlock`
  - `Brushes_ImageBrushChangingCornerRadius_Uno_UI_Samples_UITests_ImageBrushTestControl_ImageBrushChangingCornerRadius`
  - `Icons_UITests_Shared_Windows_UI_Xaml_Controls_BitmapIconTests_BitmapIcon_Foreground_UITests_Shared_Windows_UI_Xaml_Controls_BitmapIconTests_BitmapIcon_Foreground`
  - `Animations_DoubleAnimation_opacity_final_state_UITests_Windows_UI_Xaml_Media_Animation_DoubleAnimation_FinalState_Opacity`
  - `ComboBox_ComboBox_ComboBoxItem_Selection_UITests_Shared_Windows_UI_Xaml_Controls_ComboBox_ComboBox_ComboBoxItem_Selection`
  - `ComboBox_ComboBox_Picker_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ComboBox_ComboBox_Picker`
  - `MUX_NumberBox_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `Animations_UITests_Windows_UI_Xaml_Media_Animation_ColorAnimation_Background_UITests_Windows_UI_Xaml_Media_Animation_ColorAnimation_Background`
  </details>

* `ios`: **6** changed over 255

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Validate_Offscreen_Shapes_UITests_Windows_UI_Xaml_Shapes_Offscreen_Shapes`
  - `ProgressRing_Visibility_Collapsed_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `ListView_ListViewVariableItemHeightLong_InitializesTest_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewVariableItemHeightLong`
  - `TextBox_UpdatedBinding_On_OneWay_Mode_UITests_Windows_UI_Xaml_Controls_TextBox_TextBox_Bindings`
  - `When_IsEnabled_Initially_False_And_Inherited_UITests_Shared_Windows_UI_Xaml_Controls_ControlTests_Control_IsEnabled_Inheritance`
  - `ProgressRing_IsEnabled_Running_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  </details>

* `ios-Snap`: **53** changed over 992

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Brushes_ImageBrushLocalAsset_Uno_UI_Samples_UITests_ImageBrushTestControl_ImageBrushLocalAsset`
  - `NavigationView_MUXControlsTestApp_NavigationViewRS4Page_MUXControlsTestApp_NavigationViewRS4Page`
  - `Pickers_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming`
  - `TextBlock_TextBlockMultilineInStarStackPanel_Uno_UI_Samples_Content_UITests_TextBlockControl_TextBlockMultilineInStarStackPanel`
  - `Brushes_Uno_UI_Samples_Samples_Shared_Content_UITests_ImageBrushInList_Uno_UI_Samples_Samples_Shared_Content_UITests_ImageBrushInList`
  - `SwipeControl_MUXControlsTestApp_SwipeControlPage2_MUXControlsTestApp_SwipeControlPage2`
  - `Brushes_ImageBrushStretch2_Uno_UI_Samples_UITests_ImageBrushTestControl_ImageBrushStretch2`
  - `Image_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Horizontal_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Horizontal`
  - `Image_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Vertical_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Vertical`
  - `Microsoft_UI_Xaml_Media_UITests_Windows_UI_Xaml_Media_ThemeShadowTests_ThemeShadow_Basic_UITests_Windows_UI_Xaml_Media_ThemeShadowTests_ThemeShadow_Basic`
  - `Microsoft_UI_Xaml_Media_UITests_Windows_UI_Xaml_Media_ThemeShadowTests_ThemeShadow_Overlap_UITests_Windows_UI_Xaml_Media_ThemeShadowTests_ThemeShadow_Overlap`
  - `Brushes_BorderImageBrush_Uno_UI_Samples_UITests_ImageBrushTestControl_BorderImageBrush`
  - `ColorPicker_WinUIColorPickerPage_UITests_Shared_Microsoft_UI_Xaml_Controls_ColorPickerTests_WinUIColorPickerPage`
  - `NavigationView_FluentStyle_NavigationViewSample_SamplesApp_Samples_Microsoft_UI_Xaml_Controls_NavigationViewTests_FluentStyle_FluentStyle_NavigationViewSample`
  - `Performance_UITests_Windows_UI_Xaml_Performance_Performance_1000ButtonsContinuousRendering_UITests_Windows_UI_Xaml_Performance_Performance_1000ButtonsContinuousRendering`
  - `GridView_GenericApp_Views_Samples_Shared_Content_UITests_GridViewGrouped_GenericApp_Views_Samples_Shared_Content_UITests_GridViewGrouped`
  - `Grid_CenteredGridinGridwiththreefixedsizechildren_Uno_UI_Samples_Content_UITests_GridTestsControl_CenteredGridinGridwiththreefixedsizechildren`
  - `ListView_ListView_With_ListViews_Count_Measure_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_With_ListViews_Count_Measure`
  - `TextBlock_UITests_Windows_UI_Xaml_Controls_TextBlockControl_TextBlock_RelativeTextSize_UITests_Windows_UI_Xaml_Controls_TextBlockControl_TextBlock_RelativeTextSize`
  - `Icons_UITests_Shared_Windows_UI_Xaml_Controls_BitmapIconTests_BitmapIcon_Foreground_UITests_Shared_Windows_UI_Xaml_Controls_BitmapIconTests_BitmapIcon_Foreground`
  </details>

* `skia-linux-screenshots`: **81** changed over 2282

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Gamepad_CurrentReading.png-dark`
  - `Gamepad_CurrentReading.png`
  - `ClipboardTests.png-dark`
  - `Examples.png-dark`
  - `Examples.png`
  - `ExpanderColorValidationPage.png-dark`
  - `ExpanderColorValidationPage.png`
  - `DropDownButtonPage.png-dark`
  - `DropDownButtonPage.png`
  - `Gamepad_Enumeration.png-dark`
  - `Gamepad_Enumeration.png`
  - `CalendarView_Theming.png-dark`
  - `CalendarView_Theming.png`
  - `ImageIconPage.png-dark`
  - `ImageBrushInList.png-dark`
  - `ImageBrushInList.png`
  - `DragDrop_TestPage.png-dark`
  - `DragDrop_TestPage.png`
  - `Focus_FocusVisual_Properties.png-dark`
  - `Focus_FocusVisual_Properties.png`
  </details>

* `skia-windows-screenshots`: **290** changed over 2282

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Border_With_ScaleTransform.png-dark`
  - `Border_With_ScaleTransform.png`
  - `DoubleAnimation_TranslateX.png-dark`
  - `DoubleAnimation_TranslateX.png`
  - `DoubleAnimation_VisualStates.png-dark`
  - `DoubleAnimation_VisualStates.png`
  - `GridWithColumnSpan.png-dark`
  - `GridWithColumnSpan.png`
  - `Grid_Auto_Center_Cell.png-dark`
  - `Grid_Auto_Center_Cell.png`
  - `Grid_Style_Local_Override.png-dark`
  - `Grid_Style_Local_Override.png`
  - `Grid_Two_bottom_row_Auto__middle_col_auto.png-dark`
  - `Grid_Two_bottom_row_Auto__middle_col_auto.png`
  - `Grid_with_attributed_string.png-dark`
  - `Grid_with_attributed_string.png`
  - `Grid_with_Fixed_Size.png-dark`
  - `Grid_with_Fixed_Size.png`
  - `Grid_with_MinWidth_MinHeight.png-dark`
  - `Grid_with_MinWidth_MinHeight.png`
  </details>

* `wasm`: **75** changed over 1057

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `SamplesApp.Wasm.Windows_UI_Xaml_Controls.ComboBox.ComboBox_Corners`
  - `SamplesApp.Wasm.Windows_UI_Xaml_Controls.ListView.ListView_IsSelected`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.ListViewIncrementalLoading`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_Padding_Unconstrained`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ComboBox.ComboBox_MaxDropdownHeight`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ComboBox.ComboBox_SelectedIndex`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MenuFlyoutTests.MenuFlyoutItem_Click`
  - `UITests.Shared.Windows_UI_Xaml_Media_Animation.SetTargetProperty`
  - `UITests.Windows_UI_Xaml.FocusTests.Focus_FocusState`
  - `UITests.Windows_UI_Xaml.FocusTests.Focus_FocusVisual_Properties`
  - `UITests.Windows_UI_Xaml_Media_Animation.ColorAnimation_Background`
  - `Uno.UI.Samples.Content.UITests.ButtonTestsControl.Custom_Button_With_ContentTemplate_And_StackPanel`
  - `Uno.UI.Samples.Content.UITests.ButtonTestsControl.RadioButton_Multiple_Unnamed_Groups`
  - `Uno.UI.Samples.Content.UITests.ButtonTestsControl.RadioButton_With_GroupName`
  - `UITests.Microsoft_UI_Xaml_Controls.NumberBoxTests.NumberBox_ExpressionTest`
  - `UITests.Microsoft_UI_Xaml_Controls.NumberBoxTests.NumberBox_Header`
  - `UITests.Shared.Windows_UI_Xaml_Controls.CommandBar.CommandBar_Xaml_Automated`
  - `UITests.Windows_UI_Xaml_Input.Keyboard.Keyboard_Events`
  - `UITests.Windows_UI_Xaml_Media.AcrylicBrushTests.BasicAcrylicBrushTest`
  - `UITests.Shared.Windows_UI_Xaml.UIElementTests.TransformToVisual_Transform`
  </details>

* `wasm-automated-net9.0-WinUI-Benchmarks-automated`: **0** changed over 1

* `wasm-automated-net9.0-WinUI-Default-automated`: **20** changed over 877

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `SelectionContentTest_UITests_Microsoft_UI_Xaml_Controls_TabViewTests_TabViewItemsSourceTests`
  - `Default_StrokeThickness_MyRect`
  - `TestProgressRing_InitialState_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_WinUIProgressRing_Features`
  - `Default_StrokeThickness_MyPolygon`
  - `Default_StrokeThickness_MyLine`
  - `UpDownEnabledTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `UpDownTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `When_NoSelection_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `ListView_SelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `MinMaxTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `Default_StrokeThickness_MyPath`
  - `BasicExpressionTest_UITests_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBox_ExpressionTest`
  - `When_MultipleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `SequentialAnimations_SamplesApp_Windows_UI_Xaml_Media_Animation_SequentialAnimationsPage`
  - `When_StretchAndAlignmentNone_ImageBrush-50-50-None-XLeft-YBottom`
  - `When_Theme_Changed_No_Crash_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming`
  - `DecimalFormatterTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `Default_StrokeThickness_MyEllipse`
  - `When_NoSelectionWithItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_SingleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  </details>

* `wasm-automated-net9.0-WinUI-RuntimeTests-0`: **0** changed over 1

* `wasm-automated-net9.0-WinUI-RuntimeTests-1`: **0** changed over 1

* `wasm-automated-net9.0-WinUI-RuntimeTests-2`: **0** changed over 1

</details>
